packages/client/src/pages/admin_preferences/PreviewField.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { useFormikContext } from "formik";
+import { EmailTemplate, interpolateText } from "@eisbuk/shared";
+import { FormField, FormFieldVariant } from "@eisbuk/ui";
+import { useTranslation, EmailTemplateLabel } from "@eisbuk/translations";
+import { replaceHTMLTags } from "@/utils/helpers";
+
+export interface PreviewFieldProps {
+  template: EmailTemplate;
+  name: string;
+}
+const PreviewField: React.FC<PreviewFieldProps> = ({ name, ...props }) => {
+  const {
+    values: { emailTemplates },
+    setFieldValue,
+  } = useFormikContext<any>();
+
+  const { t } = useTranslation();
+
+  const subject = replaceHTMLTags(emailTemplates[name].subject);
+  const html = replaceHTMLTags(emailTemplates[name].html);
+  React.useEffect(() => {
+    if (emailTemplates[name].subject.trim() !== "") {
+      setFieldValue(
+        `${name}-subject-preview`,
+        `${interpolateText(subject, {
+          organizationName: "Display Name",
+          name: "Saul",
+          surname: "Goodman",
+          bookingsLink: "saulGoodman.com",
+          bookingsMonth: "april",
+          extendedBookingsDate: "02/01",
+          icsFile: "icsFile.ics",
```

Comment:
Can we lift this to the out of the component as e.g `const previewDefaults` and update the organizationName default value from "Display Name" to "Organisation Name"